bigchaindb/pipelines/election.py

```diff
@@ -50,10 +52,6 @@ def requeue_transactions(self, invalid_block):
         return invalid_block
 
 
-def get_changefeed():
-    return ChangeFeed(table='votes', operation=ChangeFeed.INSERT)
```

Comment:
For the sake of testing (in the tests below), I could see the usefulness of keeping these as separate functions (in each pipeline) to avoid duplicating how we get a pipeline's changefeed later.